src/tests/e2etest/ZtsiTests.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using Microsoft.Azure.Devices.Shared;
+using Microsoft.Azure.Devices;
+using System.Text.Json;
+using System;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+
+namespace e2etesting
```

Comment:
>e2etesting [](http://example.com/codeflow?start=10&length=10)

Please use for namespace same name convention. Camel casing, etc. In this case: "E2eTesting" for example

Reply:
Thanks for pointing this out. I have fixed this issue.